app/bin/executor.py

```diff
@@ -34,6 +35,7 @@
     "instance_name", f"latigo-executor-{latigo_version}-{socket.getfqdn()}"
 )
 threading.current_thread().name = instance_name
+add_azure_logging(logger, config.get("executor", {}).get("azure_monitor_logging_enabled"), config.get("executor", {}).get("azure_monitor_instrumentation_key"))
```

Comment:
Please make a function to get Azure params. It's reused in both scheduler and executor.
Maybe smth like:
```suggestion
import Dist from typing #(move this to the top imports)

def get_nested_config_value(dictionary: Dict, *keys):  # <- this func should be moved to one separate file
    return reduce(lambda d, key: d.get(key) if d else None, keys, dictionary)

is_azure_log_enabled = get_nested_config_value(config, "scheduler", "azure_monitor_logging_enabled")
azure_monitor_key = get_nested_config_value(config, "scheduler", "azure_monitor_instrumentation_key") 

add_azure_logging(is_azure_log_enabled, azure_monitor_key)
```